HyperVExtension/BuildDevSetupAgentHelper.ps1

```diff
@@ -80,6 +80,7 @@ Try {
 
   $msbuildArgs = @(
       ("HyperVExtension\DevSetupAgent.sln"),
+      ("/m"),

```

Comment:
I've always ran into problems building with `/m` on the official builds.  But, if it works here.  Cool.

Reply:
Seems to work in official builds, its a shame its not compatible with WAP Project